### ISSUE TITLE
Bug - Cron.sh - grep: Warning: spare \ before ~

### ISSUE THREAD

**Issue by tgurr** [znuny/Znuny#326]
### Environment

- OS: Linux
- Browser: Any
- Znuny version: 6.0.48 (but the [line](https://github.com/znuny/Znuny/blob/dev/bin/Cron.sh#L67) in question appears to be identical in current git master)

### Expected behaviour

No warning about deprecated behaviour reported on running `su -c "/opt/otrs/bin/Cron.sh start" -s /bin/bash otrs`.

### Actual behaviour

```
# su -c "/opt/otrs/bin/Cron.sh start" -s /bin/bash otrs
grep: Warnung: überzähliges \ vor ~
(using /opt/otrs) done
```

### How to reproduce

1. have the current `grep (GNU grep) 3.8` installed
2. run `su -c "/opt/otrs/bin/Cron.sh start" -s /bin/bash otrs`

### Additional information

I didn't prepare a pull request just removing the \ before ~ as the warning suggests as I'm not a grep/regex expert to verify that it's still doing what it is supposed to do.

### Screenshots
none

**Comment by hanneshal** [comment]
Hi @tgurr,

never seen this before. Which OS are you using?

Thanks

**Comment by tgurr** [comment]
> never seen this before. Which OS are you using?

Gentoo Linux, Exherbo Linux, but of course this affects any distribution shipping grep >= 3.8, you can reproduce it with running e.g. `ls -d * | grep -Ev "(\.(dist|rpm|bak|backup|custom_backup|save|swp)|\~)$"` through grep >= 3.8.

`NEWS` file of 3.8 mentions:

```
Regular expressions with stray backslashes now cause warnings, as
their unspecified behavior can lead to unexpected results.
For example, '\a' and 'a' are not always equivalent
<https://bugs.gnu.org/39678>.  Similarly, regular expressions or
subexpressions that start with a repetition operator now also cause
warnings due to their unspecified behavior; for example, *a(+b|{1}c)
now has three reasons to warn.  The warnings are intended as a
transition aid; they are likely to be errors in future releases.
```

**Comment by hanneshal** [comment]
Thx for the update.
We get on to it. 🎄

**Comment by jepf** [comment]
Fixed for release 6.5.1 with 091a1a112050227bb79696a3cc70092f18eab621
